### ISSUE TITLE
Installing Julia uses too many processes in parallel

### ISSUE THREAD

**Issue by eschnett** [JuliaLang/julia#20763]
I want to install Julia 0.5 on Comet at SDSC, starting from the "full" tarball. This builds OpenBLAS and runs the OpenBLAS test cases, which fail with this error message:
```
 Test of subprogram number 10            IZAMAX
                                    ----- PASS -----
OpenBLAS blas_thread_init: pthread_create: Resource temporarily unavailable
OpenBLAS blas_thread_init: RLIMIT_NPROC 1024 current, 2048 max
make[3]: *** [run_test] Interrupt
make[2]: *** [tests] Error 2
\033[33;1m*** Clean the OpenBLAS build with 'make -C deps clean-openblas'. Rebuild with 'make OPENBLAS_USE_THREAD=0' if OpenBLAS had trouble linking libpthread.so, and with 'make OPENBLAS_TARGET_ARCH=NEHALEM' if there were errors building SandyBridge support. Both these options can also be used simultaneously. ***\033[0m
make[1]: *** [build/openblas-12ab1804b6ebcd38b26960d65d254314d8bc33d6/libopenblas64_.so] Error 1
make: *** [julia-deps] Error 2
```

I started the build via a `make -j4`.

Comet has a rather strict limit on the number of concurrent user processes (1024), and OpenBLAS apparently exceeds this when running test cases as it tries to run many threads in parallel.

It would be good if Julia ran the OpenBLAS tests in such a way that the limit I chose when running make (`-j4`) is respected, at least approximately.

**Comment by tkelman** [comment]
unless there's already an option for this we can use, this is probably an openblas issue rather than a julia one

**Comment by Sacha0** [comment]
#14807 related? Best!

**Comment by ViralBShah** [comment]
Maybe the solution is just to do `make` without concurrency on such a system?

**Comment by ViralBShah** [comment]
I really don't think openblas starts so many threads. I am going to close this one and perhaps reopen if it is still an issue, now that we merged openblas 0.3.

**Comment by ViralBShah** [comment]
Also does appear to be a dup of #14807
